tests/checkpoint/test_checkpoint.py

```diff
@@ -1580,33 +1576,6 @@ def test_newstyle_checkpoint_raises_error_if_validator_specified_in_constructor_
     )
 
 
-@pytest.mark.unit
-def test_newstyle_checkpoint_raises_error_if_validator_specified_in_constructor_and_validator_is_specified_in_run(
```

Comment:
@Kilo59 Is this test being deleted?  If so, which one does is seem to be the duplicate of?  I would like to review these proposed deletions meticulously, because we have tests that look similar, but are actually distinct in subtle yet important ways.  If, however, it is indeed a duplicate (as I have found and deleted a few), then it should be deleted.  Thank you.

Reply:
I'm going to save this for another PR. but yes, these do appear to be duplicates.